src/app/zap-templates/zcl/data-model/chip/media-playback-cluster.xml

```diff
@@ -26,134 +26,84 @@ limitations under the License.
-      <description>Upon receipt, this SHALL cause the handler to be invoked for "Previous". User experience is context-specific. This will often Go back to the previous media playback item.</description>
-      <arg name="mediaPlaybackStatus" type="MediaPlaybackStatus"/>
-    </command>
-
-    <command source="server" code="0x05" name="MediaNextResponse" optional="false" disableDefaultResponse="true">
-      <description>Upon receipt, this SHALL cause the handler to be invoked for "Next". User experience is context-specific. This will often Go forward to the next media playback item.</description>
-      <arg name="mediaPlaybackStatus" type="MediaPlaybackStatus"/>
-    </command>
-
-    <command source="server" code="0x06" name="MediaRewindResponse" optional="false" disableDefaultResponse="true">
-      <description>Upon receipt, this SHALL Rewind through media. Different Rewind speeds can be used on the TV based upon the number of sequential calls to this function. This is to avoid needing to define every speed now (multiple fast, slow motion, etc).</description>
-      <arg name="mediaPlaybackStatus" type="MediaPlaybackStatus"/>
-    </command>
-
-    <command source="server" code="0x07" name="MediaFastForwardResponse" optional="false" disableDefaultResponse="true">
-      <description>Upon receipt, this SHALL Advance through media. Different FF speeds can be used on the TV based upon the number of sequential calls to this function. This is to avoid needing to define every speed now (multiple fast, slow motion, etc).</description>
-      <arg name="mediaPlaybackStatus" type="MediaPlaybackStatus"/>
-    </command>
-
-    <command source="server" code="0x08" name="MediaSkipForwardResponse" optional="false" disableDefaultResponse="true">
-      <description>Upon receipt, this SHALL Skip forward in the media by the given number of seconds, using the data as follows:</description>
-      <arg name="mediaPlaybackStatus" type="MediaPlaybackStatus"/>
-    </command>
-
-    <command source="server" code="0x09" name="MediaSkipBackwardResponse" optional="false" disableDefaultResponse="true">
-      <description>Upon receipt, this SHALL Skip backward in the media by the given number of seconds, using the data as follows:</description>
-      <arg name="mediaPlaybackStatus" type="MediaPlaybackStatus"/>
-    </command>
-
-    <command source="server" code="0x0B" name="MediaSeekResponse" optional="false" disableDefaultResponse="true">
-      <description>Upon receipt, this SHALL change the playback position in the media to the given position using data as follows:</description>
-      <arg name="mediaPlaybackStatus" type="MediaPlaybackStatus"/>
+    <command source="server" code="0x0A" name="PlaybackResponse" optional="false">
+      <description>This command SHALL be generated in response to various Playback Request commands.</description>
+      <arg name="statusEnum" type="StatusEnum"/>
```

Comment:
This field is named "Status" in the spec.

Reply:
Updated.